salt/metalk8s/defaults.yaml

```diff
@@ -1,3 +1,32 @@
+kubeadm_preflight:
+  mandatory:
+    packages:
+      - util-linux            # provides nsenter, mount
+      - iproute               # provides ip
+      - iptables              # provides iptables
+    ports:
+      - 10250
+      - 10251
+      - 10252
+      - 2379
+      - 2380
+    services:
+      running:
+        - containerd
+    sysctl_values:
+      net.bridge.bridge-nf-call-ip6tables: 1
+      net.bridge.bridge-nf-call-iptables: 1
+      net.ipv4.ip_forward: 1
+  recommended:
+    packages:
+      - ebtables              # provides ebtables
+      - ethtool               # provides ethtool
+      - socat                 # provides socat
+      - iproute               # provides tc
+      - coreutils             # provides touch
+    services:
+      enabled:
+        - kubelet
```

Comment:
I think it's not a good idea to enable kubelet here as it's can't start at this point it's more kubelet-start who will enable and start it.
For me we should just check that kubelet is installed, test.fail if not